test/acceptance/features/createTodo.feature

```diff
@@ -0,0 +1,27 @@
+Feature: createTodo
+  As a user
+  I want to create a todo
+  So that I would keep track of my todo
+
+  Background:
+    Given user has browsed to the homepage
+
+  Scenario: user should be able to create a todo entering both subject and content
+    When the user creates a todo entering subject "subject" and content "content"
+    Then a new todo with subject "subject" and content "content" should be visible
+
+  Scenario: user should not be able to create a todo entering subject only
+    When the user creates a todo entering subject "subject" only
+    Then an error with error message "Please enter content of your todo!" should be visible
+
+  Scenario: user should not be able to create a todo entering content only
+    When the user creates a todo entering content "content" only
+    Then an error with error-message "Please enter a subject!" should be visible
+
+  Scenario: user should not be able to create a blank todo
+    When the user creates a blank todo
+    Then an error with error-message "Please enter a subject!" should be visible
```

Comment:
```suggestion
    Then an error with error-message "Please enter a subject!" should be visible
```
This step looks similar to `Then an error with error message "Please enter content of your todo!" should be visible
`, Please fix this and make it more readable